### DISCUSSION TITLE
Can I destroy the store including the state object in multiple store?

### DISCUSSION THREAD

**Discussion by dong-ho-lee** [pmndrs/zustand#1429] [Q&A]
In react-native, I am in an environment where each view uses a store. I want to destroy the store when the view disappears. StoreApi's destroy is called, but individual instances are identified when viewed in redux dev tool. Looking at the docs, it seems that the destroy function simply serves to release the listeners of the subscription. Is there any way?

**Comment by dai-shi** [comment]
Yeah, and that `destroy` method will be removed in the future.

I think you can just clear the state?
```js
useSomeStore.setState(null)
```

**Reply by dong-ho-lee** [reply]
thank you🙇🏽 I will try it!
